demos/api/core_web_vitals/main.js

```diff
@@ -0,0 +1,182 @@
+    }
+    process.exit(1);
+  }
+  return json;
+};
+
+/**
+ * Utility method to transform the response's metric data into an array of usable metric objects
+ * Example return value:
+  [
+    {
+      "acronym": "FCP",
+      "name": "first_contentful_paint",
+      "labeledBins": [{
+        "label": "good",
+        "percentage": 43.51,
+        "start": 0,
+        "end": 1000,
+        "density": 0.4351
+      }, // ... (other bins) ]
+      "percentiles": {
+          "p75": 1792
+      },
+    }, // ... (other metrics) ]
+*
+* @return {{acronym: string, name: string, labelsBins: Array<{label: 'good'|'needs improvement'|'poor', percentage: number, start: number, end?: number, density: number}>}}*/
+function labelMetricData(metrics) {
+  const nameToAcronymMap = {
+    first_contentful_paint: 'FCP',
+    largest_contentful_paint: 'LCP',
+    first_input_delay: 'FID',
+    cumulative_layout_shift: 'CLS',
+  };
+
+  return Object.entries(metrics).map(([metricName, metricData]) => {
+    const standardBinLabels = ['good', 'needs improvement', 'poor'];
+    const metricBins = metricData.histogram;
+
+    // We assume there are 3 histogram bins and they're in order of: good => poor
+    const labeledBins = metricBins.map((bin, i) => {
+      // Assign a good/poor label, calculate a percentage, and add retain all existing bin properties
+      return {
+        label: standardBinLabels[i],
+        percentage: bin.density * 100,
+        ...bin,
+      };
+    });
+
+    return {
+      acronym: nameToAcronymMap[metricName],
+      name: metricName,
+      labeledBins,
+      percentiles: {
+          p75: metricData.percentiles.p75,
+      },
+    };
+  });
+}
+
+// Gather the data and display it.
+(async function () {
+  const json = await CrUXApiUtil.query(request);
+
+  const labeledMetrics = await labelMetricData(json.record.metrics);
+
+  // The CWV 75%ile thresholds.
+  const nameToCWVMap = {
+    first_contentful_paint: 2500,
+    first_input_delay: 100,
+    cumulative_layout_shift: '0.1',
+  };
+
+  // Display metric results
+  let resId = '';
+  let identifier = '';
+  if (json.record.key.origin) {
+    resId += 'Origin: ' + json.record.key.origin;
+    identifier = json.record.key.origin;
+  } else {
+    resId += 'Url: ' + json.record.key.url;
+    identifier = json.record.key.url;
+  }
+  if (json.record.key.formFactor) {
+    resId += ', FormFactor: ' + json.record.key.formFactor;
+  }
+  // Add form factor too...
+  console.log('\n' + resId);
+  let pass = true;
+  for (const metric of labeledMetrics) {
+    // console.log("Metric: " + JSON.stringify(metric, null, 2));
+    if (nameToCWVMap[metric.name]) {
+        // Check pass fail
+        if (metric.percentiles.p75 <= nameToCWVMap[metric.name]) {
+            console.log('Metric "' + metric.name + '" ' + chalk.green('passes') + ' the Core Web Vitals assessment. (' + metric.percentiles.p75 + ' <= ' + nameToCWVMap[metric.name] + ')');
```

Comment:
These could really benefit from string templates

```suggestion
            console.log(`Metric "${metric.name}" ${chalk.green('passes')} the Core Web Vitals assessment. (${metric.percentiles.p75} <= ${nameToCWVMap[metric.name]})`);
```